### PR TITLE
featured artists now always positioned below summary

### PR DESCRIPTION
**Pull request by mbilokonsky**
This implements [GROW-1316](https://artsyproduct.atlassian.net/browse/GROW-1316). Easy enough, removed the responsive positioning logic and the module now sits in the same place regardless of screen size. @xtina-starr tagging you for PR #3 today, woo! :)

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit ab19cad6388b2564e0f6fe139426e35d174885ff

https://deploy-preview-2564--artsy-reaction.netlify.com

File: src/Apps/Collect/Components/Collection/Header/index.tsx [hunk 1]
```diff
@@ -278,10 +278,10 @@ export const CollectionHeader: FC<Props> = ({ artworks, collection }) => {
                     </Col>
                     <Col
                       sm={12}
-                      md={isColumnLayout ? "12" : "3"}
-                      mdOffset={isColumnLayout ? null : 1}
-                      lgOffset={isColumnLayout ? null : 1}
-                      xlOffset={isColumnLayout ? null : 1}
+                      md={12}
+                      mdOffset={null}
```
Comment (xtina-starr) [inline src/Apps/Collect/Components/Collection/Header/index.tsx]: Sweet, we could probably remove the `mdOffset`, `lgOffset `, & `xlOffset ` all together.

Comment (mbilokonsky) [reply src/Apps/Collect/Components/Collection/Header/index.tsx]: Good call, done!

**Comment by artsyit** [comment]
:rocket: PR was released in v16.17.23 :rocket:

**Pull request merged**